### PR TITLE
Package & DropdownMenu position

### PR DESCRIPTION
**Pull request by Duncid**

### CODE REVIEW

File: sparkle/src/components/DropdownMenu.tsx [hunk 1]
```diff
@@ -8,6 +16,10 @@ import { Icon } from "./Icon";
 import { Item as StandardItem } from "./Item";
 import { Tooltip, TooltipProps } from "./Tooltip";
 
+const ButtonRefContext = React.createContext<MutableRefObject<any> | null>(
```
Comment (spolu) [inline sparkle/src/components/DropdownMenu.tsx:19]: `any` is not allowed :)

**Review by spolu — COMMENTED** [review]
Would be awesome if we can get rid of that any

File: sparkle/src/components/DropdownMenu.tsx (hunk 1 above)
Comment (Duncid) [reply sparkle/src/components/DropdownMenu.tsx:19]: How would you solve it?
We do want it to take any in that case

Comment (spolu) [reply sparkle/src/components/DropdownMenu.tsx:19]: There's likely a generic type that covers all we need (HTML something or React component smthg)

**Pull request merged**